README.md

```diff
@@ -155,6 +155,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Octions - GitHub Actions for GitHub REST API](https://github.com/maxkomarychev/octions)
 - [GitHub Actions for Docker](https://github.com/docker/github-actions)
 - [GitHub Actions for AWS](https://github.com/clowdhaus/aws-github-actions)
+- [GitHub Actions for Lazarus/FPC](https://github.com/gcarreno/setup-lazarus)
```

Comment:
Since this is not a collection of actions, it does not belong in this section. Please move this to the Utility section for now unless you see another category that is more suiting.

Reply:
Will do!